index.html

```diff
@@ -17,10 +17,53 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1>Code Journal</h1>
+          </div>
+        </div>
+      </div>
     </header>
+
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div class="row">
+        <div class="column-full">
+          <h2>New Entry</h2>
+        </div>
+      </div>
+
+      <div data-view="entry-form">
```

Comment:
Lines 30-34 need to be inside the `div` with the `data-view="entry-form"`, because eventually we will implement view swapping, and we only want the "New Entry" text to appear when we are on the entry-form view